src/app/components/mnoe-api/app-instances.svc.coffee

```diff
@@ -33,14 +35,16 @@ angular.module 'mnoEnterpriseAngular'
       fetchAppInstances()
 
     # Retrieve app instances from the backend
-    fetchAppInstances = ->
+    # If unscoped, retrieve app instances irrespective of tenant
+    fetchAppInstances = (unscoped) ->
       # Workaround as the API is not standard (return a hash map not an array)
       # (Prefix operation by '/' to avoid data extraction)
       # TODO: Standard API
       defer = $q.defer()
       MnoeOrganizations.get().then(
         ->
-          _self.appInstancesPromise = MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances').get().then(
+          query = if unscoped == true then '?unscoped=true' else ''
+          _self.appInstancesPromise = MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances' + query).get().then(
```

Comment:
For query parameters, [you can pass an object to `get()`](https://github.com/mgonto/restangular#element-methods):
```coffeescript
params = if unscoped then {unscoped: true} else {}
xxx.one('/app_instances').get(params)
```